server/djangoapp/templates/djangoapp/index.html

```diff
@@ -17,7 +17,11 @@
     <!-- Remove this line the first time you edit this file -->
      This is the index page of your Django app!
     <!--Add a nav bar here -->
-
+    <nav>
```

Comment:
Nice work @Ajay-Dhangar but this pull request should be made on your branch. 
Meanwhile, your href value for line 21, 22 and 23 should be the url addresses you created in your urls.py not the html file
e.g `/djangoapp/`, `/djangoapp/about/` and `/djangoapp/contact/`